alchemist-web-renderer/build.gradle.kts

```diff
@@ -90,35 +89,24 @@ application {
 }
 
 tasks.getByName<JavaExec>("run") {
-    classpath(tasks.getByName<Jar>("shadowJar"))
+    val shadowJarTask = tasks.getByName<Jar>("shadowJar")
+    dependsOn(shadowJarTask)
```

Comment:
This seems to imply that I need to pack a fatJar in order to run the application -- why so?

Reply:
The [Kotlin Multiplatform Full Stack App example](https://kotlinlang.org/docs/multiplatform-full-stack-app.html) uses this trick to have acces to all the compiled files and resources, even the javascript one, simplifiyng the build. I can try to use the output of compile tasks though, that should work too and I think I can do it. I'm gonna update this one I found out.

Reply:
I cannot find where. Looks like the opposite from that page, the `jvmJar` task depends on a `webpackTask`.
Here we are saying that, in order to run, we need to pack a fatJar (and test it, since there is a finalizer).

Or do you refer to:
```kotlin
tasks.getByName<JavaExec>("run") {
    classpath(tasks.getByName<Jar>("jvmJar")) // so that the JS artifacts generated by `jvmJar` can be found and served
}
```

?